src/Handlers/BlockWorkerHandler.cs

```
@@ -0,0 +1,21 @@
+public class BlockWorkerHandler
+{
+    private readonly WorkerRepository _workerRepository;
+
+    public BlockWorkerHandler(WorkerRepository workerRepository)
+    {
+        _workerRepository = workerRepository;
+    }
+
+    public async Task<IActionResult> HandleAsync(BlockWorkerRequest request)
+    {
+        try
+        {
+            _workerRepository.BlockWorker(request.WorkerId, request.FacilityId);
+        }
+        catch (Exception e)
+        {
+            return new JsonResult(new { error = e.Message });
```

If I’m not mistaken, JsonResult returns a 200 OK status, and maybe we should return an error code since this is inside a catch block. What do you think?